src/components/DetailsResults/DetailsResults.js

```diff
@@ -36,11 +36,11 @@ import './detailsResults.scss'
 
 const DetailsResults = ({ allowSortBy, defaultSortBy, defaultDirection, excludeSortBy, job }) => {
   const tableHeaders = useMemo(() => {
-    return isEmpty(job.error) ? resultsTableHeaders(job) : []
+    return job.iterationStats.length ? resultsTableHeaders(job) : []
```

Comment:
Looks like there is no need in this condition. You can just have:
```suggestion
    return resultsTableHeaders(job)
```
and for tableContent: 
```suggestion
    return resultsTableContent(job)
```